website/source/docs/agent/config.html.md

```diff
@@ -391,6 +391,16 @@ configured on client nodes.
     join any nodes when it starts up. Addresses can be given as an IP, a domain
     name, or an IP:Port pair. If the port isn't specified the default Serf port,
     4648, is used.  DNS names may also be used.
+  * <a id="encrypt">`encrypt`</a> Specifies the secret key to use for encryption
+    of Nomad server's gossip network traffic. This key must be 16-bytes that are
```

Comment:
or 24 or 32? Or do we not even want to bother mentioning the aes 192 & 256 support?